src/data/dfinsupp.lean

```diff
@@ -982,4 +1010,36 @@ lemma dfinsupp_prod_apply [monoid R] [comm_monoid S]
 
 end monoid_hom
 
+namespace add_monoid_hom
+variables {R S : Type*}
+
+/-! The above lemmas, repeated for `dfinsupp.sum_add_hom`. -/
+@[simp]
+lemma map_dfinsupp_sum_add_hom
+  [add_comm_monoid R] [add_comm_monoid S]
+  [Π (i : ι), add_comm_monoid (β i)]
```

Comment:
```suggestion
  [add_comm_monoid R] [add_comm_monoid S] [Π (i : ι), add_comm_monoid (β i)]
```
I think you can save some screen real estate by squeezing more typeclass assumptions on one line.